src/scripts/train/OLMo2-32B.py

```diff
@@ -52,20 +53,23 @@ def build_optim_config(common: CommonComponents) -> AdamWConfig:
 
 
 def build_trainer_config(common: CommonComponents) -> TrainerConfig:
+    project_name = "peteish32"
     return (
         TrainerConfig(
-            save_folder=common.save_folder,
+            save_folder=f"gs://ai2-llm/checkpoints/{project_name}/",
```

Comment:
Why change this?

Reply:
It defaults to something under my name? Not what we want for an official run?

Reply:
Especially if we swap babysitting responsibilities during the run